torch/csrc/jit/python/init.cpp

```diff
@@ -752,10 +753,31 @@ void initJITBindings(PyObject* module) {
       .def(
           "_jit_set_bailout_depth",
           [](size_t depth) {
+            TORCH_WARN("Use _jit_set_fusion_strategy, bailout depth is deprecated. Setting to (Dynamic, ", depth, ")");
             size_t old_depth = getBailoutDepth();
-            getBailoutDepth() = depth;
+            FusionStrategy strat = {{FusionBehavior::DYNAMIC, depth}};
+            setFusionStrategy(strat);
             return old_depth;
           })
+      .def("_jit_set_fusion_strategy",
+          [](std::vector<std::pair<std::string, size_t>> strategy) {
```

Comment:
should we consider adding bindings to the `STATIC` , `DYNAMIC` enum? It would simplify this code significantly and also remove the whole class of issues where a user mistypes in python `STAIC`.

Reply:
the typical thing in pytorch is to use strings as enum, and IMO this is an easier api